internal/cmd/io_binary_test.go

```diff
@@ -18,19 +18,34 @@ func Test_newBinaryCmd_Key_Value_Pair(t *testing.T) {
 
 	tests := []*config{
 		{
-			name:      "use-path",
+			name:      "Use_Path",
 			urlString: "binary:///executable?-key=value",
-			expected:  "/executable -key value",
+			expected:  "\\executable -key value",
```

Comment:
FYI, you can put Go strings in backticks to disable escape sequences. You may prefer to do that for these strings with backslashes to avoid escaping.